packages/dbml-core/__tests__/parser/postgres-parse/output/cmd_comment.out.json

```diff
@@ -7,16 +8,22 @@
           "name": "my_column",
           "type": {
             "type_name": "varchar",
-            "args": null
+            "schemaName": null
```

Comment:
Why there is `schemaName` inside a field object? Is the generated JSON correct?

Reply:
That's the name of the `type` of the column. Incase the type is an Enum, the schemaName will contain non-null value.